src/World.h

```diff
@@ -1042,7 +1042,10 @@ class cWorld :
 	void UpdateSkyDarkness(void);
 
 	/** Generates a random spawnpoint on solid land by walking chunks and finding their biomes */
-	void GenerateRandomSpawn(void);
+	void GenerateRandomSpawn(int a_MaxSpawnRadius);
+
+	/** Can the specified coordinates be used as a spawn point? */
+	bool CanSpawnAt(double a_X, double a_Z, double &a_Y);
```

Comment:
All our functions use X, Y, Z, this one should, too.
Document the Y being an out-param.